Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1100,9 +1100,14 @@
 #endif
 
 #if ENABLE(LOGD_BLOCKING_IN_WEBCONTENT)
-(allow mach-lookup (with telemetry) (global-name
-    "com.apple.logd"
-    "com.apple.logd.events"))
+(with-filter (require-entitlement "com.apple.private.disable-log-mach-ports")
+    (deny mach-lookup (with no-report) (global-name
```

Comment:
Why do we want no report of violation? Should we get the report and find out what's violating the policy from the report?